src/java.desktop/macosx/native/libsplashscreen/splashscreen_sys.m

```diff
@@ -276,6 +276,9 @@ static int isInAquaSession() {
     int rslt = pthread_attr_init(&attr);
     if (rslt != 0) return;
     rc = pthread_create(&thr, &attr, SplashScreenThread, (void *) splash);
```

Comment:
For the sake of symmetry, could you rename rc to rslt here?

Reply:
Sure, I changed it .